code/script.js

```diff
@@ -1,51 +1,117 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
+let botQuestion = 1
 
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
 
-// Declare your functions after this comment
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
-// This function will add a chat bubble in the correct place based on who the sender is
```

Comment:
It looks like you have deleted all or most of the // comments, I also started doing this to "clean up" my code but then realized that we should keep them to describe what we need to do and/or what we have done, both for ourselves and others to better understand our code.